### ISSUE TITLE
Is there a table about the version relationship between msgpack-c and boost?

### ISSUE THREAD

**Issue by ygj6** [msgpack/msgpack-c#843]
Just like this:
|  msgpack-c   | boost version  | boost modules |
| ---------- | --- | ------------------------- |
| cpp-3.0.0  | 1.x.x~1.x.x  | chrono, context , ... |
| cpp-2.0.0  | 1.x.x~1.x.x | chrono, context,  ... |
| cpp-1.0.0  | 1.x.x~1.x.x | chrono, context,  ... |

**Comment by redboltz** [comment]
No. https://github.com/msgpack/msgpack-c/blob/master/CHANGELOG.md has some information about boost type support.
